Makefile

```diff
@@ -50,4 +50,4 @@ image-manifest:
 
 .PHONY: image-scan
 image-scan:
-	trivy --severity $(SEVERITIES) --no-progress --ignore-unfixed $(ORG)/hardened-whereabouts:$(TAG)
+	trivy --severity $(SEVERITIES) --no-progress --ignore-unfixed image $(ORG)/hardened-whereabouts:$(TAG)
```

Comment:
Does this mean that `image-scan` was not working?

Reply:
yes, the update to the newer go tag was causing CI to fail because the updated trivy image requires a `scanning command`